.dockerignore

```diff
@@ -0,0 +1,6 @@
+# Ignore everything
+*
```

Comment:
I don't know if this is part of a version specific docker thing, but this does not work on my machine. No files are moved during the `COPY . .` step of docker build.

Reply:
May I know the version of your docker `docker version`

Reply:
```
> docker --version
Docker version 20.10.17, build 100c701
```

Reply:
Just to make sure, You're building it from correct context!?

Reply:
I was able to get the copy command working as you wrote it using this
```
# Ignore everything
*

# But these files...
!**/go.*
!**/*.go
```
as my dockerignore